.github/ISSUE_TEMPLATE/Feature_request.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Feature request
```

Comment:
[From what I understand](https://blog.github.com/2018-01-25-multiple-issue-and-pull-request-templates/) these will only be used if you append `?template=Feature_request.md` to the new issue URL?

If so, I'm not sure we have a use case for the specific issue templates just yet. Maybe we should just start with global issue template.

Reply:
Yes, it does change the URL but you also get this (at least, if I understand it right).

<img width="786" alt="screen shot 2018-05-14 at 8 38 23 pm" src="https://user-images.githubusercontent.com/3250117/40031822-0dd4a3ae-57b7-11e8-905d-52e62d1d6f34.png">

Reply:
BTW where did that screenshot come from?

Reply:
The SS came from webpack repo [LINK](https://github.com/webpack/webpack/issues/new/choose).

Think the global one would be nice to have? Wouldn't hurt to keep it around.

Reply:
Ya let's nuke the global issue template. The specific ones are good 'nuff.